packages/mobile/src/tokens/saga.ts

```diff
@@ -248,3 +253,23 @@ export async function getCurrencyAddress(currency: Currency) {
       return contractKit.registry.addressFor(CeloContract.StableTokenEUR)
   }
 }
+
+export function* importTokenInfo() {
+  const tokens: Token[] = yield readOnceFromFirebase('tokensInfo')
+  let token: Token
```

Comment:
nit: why declare token here instead of doing `for (const token of tokens)` in line 263?